.goreleaser.yml

```diff
@@ -25,6 +25,7 @@ builds:
       - amd64
       - arm
       - arm64
+      - ppc64le
```

Comment:
I believe we only want to build `ppc64le` for `linux` - is that correct? If so, please add entries to the `ignore` list just below to exclude the `ppc64le/darwin` and `ppc64le/windows` combinations from the build.

Reply:
entries to ignore list added